example-integration/src/main/webapp/WEB-INF/templates/fake-webshop-pending.ftl

```diff
@@ -0,0 +1,10 @@
+<#import 'macros.ftl' as macros />
+<#import 'result.ftl' as res/>
+
+<@macros.header />
+<@nav.tabComponent ''/>
+<div class="mt-3">
+    <@res.result paymentStatusMessage='Check order status here.'/>
```

Comment:
"The result of the payment is unknown. Retrieve order status for the final result."